mass/adapters/inbound/fastapi_/routes.py

```diff
@@ -15,6 +15,66 @@
 #
 
 """Contains API endpoints"""
-from fastapi import APIRouter
+from typing import Union
+
+from dependency_injector.wiring import Provide, inject
+from fastapi import APIRouter, Depends
+from fastapi.exceptions import HTTPException
+
+from mass.config import Config
+from mass.container import Container
+from mass.core import models
+from mass.ports.inbound.query_handler import (
+    ClassNotConfiguredError,
+    QueryHandlerPort,
+    SearchError,
+)
 
 router = APIRouter()
+
+
+# GET /rpc/search-options
+@router.get(
+    path="/rpc/search-options",
+    summary="Retrieve all configured resource classes and facetable properties",
+    response_model=dict[str, models.SearchableClass],
+)
+@inject
+async def search_options(
+    config: Config = Depends(Provide[Container.config]),
+) -> dict[str, models.SearchableClass]:
+    """Returns the configured searchable classes"""
```

Comment:
Maybe explain a bit more what they are and where to use them.

Reply:
Updated in [50cedb5](https://github.com/ghga-de/mass/pull/7/commits/50cedb5b5d432fcd5695b9090fdc6e82dfcab5e9)